apps/console/src/features/users/components/bulk-import-response-list.tsx

```diff
@@ -282,33 +386,56 @@ export const BulkImportResponseList: React.FunctionComponent<BulkImportResponseL
                                                 "wizardSummary.alerts.importFailed.description"
                                             }
                                             tOptions={ {
-                                                failedCount: bulkResponseSummary.failedCount
+                                                failedUserAssignmentCount: bulkResponseSummary.failedUserAssignment,
+                                                failedUserCreationCount: bulkResponseSummary.failedUserCreation
                                             } }
                                         >
-                                            Issues encountered in <b>count import(s)</b>.
+                                            Issues encountered in <b> creation operations</b>  <b> operations</b>.
                                         </Trans>
                                     </Alert>
                                 )
                     }
                 </Grid.Column>
             </Grid.Row>
-
+            <Grid.Row columns={ 1 }>
+                <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
+                    { resolveBulkResponseOperationTypeSelection() }
+                </Grid.Column>
+            </Grid.Row>
             <Grid.Row columns={ 1 }>
                 <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
                     <ListLayout
                         advancedSearch={ (
                             <AdvancedSearchWithBasicFilters
                                 onFilter={ handleUserFilter }
-                                filterAttributeOptions={ [
-                                    {
-                                        key: 0,
-                                        text: t("console:manage.features.users.advancedSearch.form.dropdown." +
-                                                "filterAttributeOptions.username"),
-                                        value: "userName"
-                                    } ] }
+                                filterAttributeOptions={
+                                    responseOperationType === BulkImportResponseOperationTypes.USER_CREATION ?
+                                        [
+                                            {
+                                                key: 0,
+                                                text: t("console:manage.features.users.advancedSearch.form.dropdown." +
+                                                    "filterAttributeOptions.username"),
+                                                value: "userName"
+                                            }
```

Comment:
Shall we move this out of return function to a constant?

Reply:
Fixed